hw/top_earlgrey/lint/chip_earlgrey_asic.waiver

```diff
@@ -63,6 +63,33 @@ waive -rules {RESET_DRIVER} -location {chip_earlgrey_asic.sv} \
 waive -rules {COMBO_LOOP} -location {chip_earlgrey_asic.sv} -regexp {'tx' driven in module 'uart_core' by 'rx' at uart_core.sv} \
       -comment "there is technically a loopback path through uart, however RX / TX should never be configured to the same pin"
 
-# Combo loops through sysrst_ctrl inversion can be ignored (TODO: need to double check, are the inversions meant to be direct?)
-waive -rules {COMBO_LOOP} -location {chip_earlgrey_asic.sv} -regexp {'cio_key2_out_o' driven in module 'sysrst_ctrl_inv' by 'key2_out_int' at sysrst_ctrl_inv.sv} \
+# Combo loops through sysrst_ctrl inversion can be ignored
+waive -rules {COMBO_LOOP} -location {chip_earlgrey_asic.sv} -regexp {'outputs\[[0-9]+\]' driven in module 'sysrst_ctrl_pin' by 'inputs\[[0-9]+\]' at sysrst_ctrl_pin} \
       -comment "sysrst_ctrl creates a feed through path directly for certain muxed pins"
+
+# External clock
+waive -rules {CLOCK_DRIVER} -location {chip_earlgrey_asic.sv}  -regexp {'(attr_padring_o|mio_attr)\[28\].pull_select' driven in module} \
+      -comment "MioPadIoc6 at index 28 may serve as an external clock input, hence the warnings"
+
+waive -rules {CLOCK_USE} -location {chip_earlgrey_asic.sv}  -msg {'mio_in_raw[28]' is used for some other purpose, and as clock 'clk_ast_ext_scn'} \
+      -comment "MioPadIoc6 at index 28 may serve as an external clock input, hence the warnings"
+
+# Unused power OK signals
+waive -rules {HIER_NET_NOT_READ NOT_READ} -location {chip_earlgrey_asic.sv} -regexp {(Signal|Net) 'ast_pwst(\.vcc_pok|_h\.io_pok|_h\.main_pok|_h\.vcc_pok)' is not read from in module 'chip_earlgrey_asic'} \
```

Comment:
do you recall which one specifically lint complained about? I think we should be a bit more specific instead of a blanket waive.

Reply:
Yeah they still show up in the [nightly](https://reports.opentitan.org/hw/chip_earlgrey_asic/lint/ascentlint/latest/results.html):

```
E   HIER_NET_NOT_READ:   chip_earlgrey_asic.sv:684        Net 'ast_pwst.vcc_pok' is not read from in module 'chip_earlgrey_asic'                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               New                                                                                                                        

E   HIER_NET_NOT_READ:   chip_earlgrey_asic.sv:685        Net 'ast_pwst_h.io_pok' is not read from in module 'chip_earlgrey_asic'                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              New                                                                                                                        

E   HIER_NET_NOT_READ:   chip_earlgrey_asic.sv:685        Net 'ast_pwst_h.main_pok' is not read from in module 'chip_earlgrey_asic'                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            New                                                                                                                        

E   HIER_NET_NOT_READ:   chip_earlgrey_asic.sv:685        Net 'ast_pwst_h.vcc_pok' is not read from in module 'chip_earlgrey_asic'                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             

```

Do these look ok? I can make them more specific.

Reply:
let me double check. Why don't you go ahead first.
I think the `pwst_h` are all fine.  `vcc_pok` i think is also okay.

Reply:
ok sounds good.